components/lwip/port/esp32/include/netdb.h

```diff
@@ -33,8 +33,17 @@
 #include "lwip/netdb.h"
 
 #ifdef ESP_PLATFORM
+
```

Comment:
I would prefer the C++ guards outside the `#ifdef`. Not very likely but there could be some platform independent code as well which should also be guarded.

Reply:
@0xjakob  You're welcome! I agree and I moved the C++ guards outside of the ifdef.